.github/workflows/meson.yml

```diff
@@ -102,3 +113,113 @@ jobs:
       - name: Dump Meson Debug Info
         if: failure()
         run: cat build/meson-logs/meson-log.txt
+
+  build-muon:
+    runs-on: ubuntu-latest
+    container:
+      image: alpine:latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        cc: [gcc]
+
+    steps:
+      - name: install deps
+        run: |
+          # muon is in edge but not yet in 3.19 (last line are muon build and test deps)
+          apk add nasm ${{ matrix.cc }} musl-dev samurai \
+                  fontconfig-dev freetype-dev fribidi-dev harfbuzz-dev \
+                  libpng-dev \
+                  git pkgconf-dev pkgconf g++
+
+      - name: build muon
+        run: |
+          git clone --depth=1 https://git.sr.ht/~lattis/muon
+          cd muon
+          sh ./bootstrap.sh build_bootstrap
+          build_bootstrap/muon setup build
+          build_bootstrap/muon -C build samu
+          build_bootstrap/muon -C build test
+          build_bootstrap/muon -C build install
+
+      - name: checkout code
+        uses: actions/checkout@v4
+
+      - name: configure
+        run: |
+          muon setup -Dtest=true build
+
+      - name: build
+        run: |
+          samu -C build
+
+      - name: check binary works
+        run: |
+          build/test/libass_test /tmp/out.png compare/test/sub1.ass 0.5
+
+      - name: lint style (informational)
+        run: |
+          # `muon fmt -q` already let’s us check _if_there’d be changes, but
```

Comment:
(typo in `lets` and missing space after `_if_`)

Reply:
fixed, thx